src/App.js

```diff
@@ -1,23 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
-
+import "./App.css";
+import ReactComponent from "./Task1/ReactComponent";
+import PureComponent from "./Task1/PureComponent";
+import FunctionComponentCounter from "./Task1/FunctionComponentAndCounter";
+import SearchForm from "./Task1/SearchForm";
+import SelectGenre from "./Task1/SelectGenre";
+import reactCreateElement from "./Task1/React.CreateElement";
```

Comment:
There is no need to create separate folders for each task. During the whole course, you will be developing one application, and at the end all of the code in this repository will be related to this app. Each task will actually continue the previous ones, so there will be links and dependencies between tasks. So splitting each task into a separate folder is also impractical too. Besides, try to imitate the real-life project development experience. In real project you wouldn't create separate folders for separate tasks.

Reply:
I fixed